offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/OfferOverviewView.groovy

```diff
@@ -201,4 +205,27 @@ class OfferOverviewView extends FormLayout {
             })
         }
     }
+
+    private static class FileNameFormatter {
+
+        /**
+         * Returns an offer file name in this schema:
+         *
+         * Q_<year>_<month>_<day>_<project-conserved-part>_<random-id-part>_v<offer-version>.pdf
+         * @param offer
+         * @return
+         */
+        static String getFileNameForOffer(Offer offer) {
+            LocalDate date = offer.modificationDate.toLocalDate()
+            String dateString = createDateString(date)
+            return "Q_${dateString}_" +
```

Comment:
do we want the Q here?
1. we always used O for the Id
2. the issues does not request the Q before the date

Reply:
We need to distinguish between ID and how the filename of the downloaded offer looks like. The filename was a specific request of the stakeholders and the implementation addresses it. I updated the issue comment to reflect the suggestions in the conversations.

Reply:
:+1: